python/tools/duplicate_structure/duplicate_structure.py

```diff
@@ -0,0 +1,259 @@
+# :coding: utf-8
+# :copyright: Copyright (c) 2024 Backlight
+
+import logging
+import argparse
+import itertools
+import time
+
+import ftrack_api
+import ftrack_api.attribute
+
+logging.basicConfig()
+
+session = ftrack_api.Session(auto_populate=False)
```

Comment:
add a comment on why initialising the session with **auto_populate** disabled

Reply:
Added.